big_scape/cli/cluster_cli.py

```diff
@@ -22,6 +23,36 @@
 @click.command()
 @common_all
 @common_cluster_query
+# binning parameters
+@click.option(
+    "--classify",
+    type=click.Choice(["none", "class", "category", "legacy"]),
+    default="class",
+    callback=validate_classify,
+    help=(
+        "Use antiSMASH/BGC 'class' or 'category' to run analyses on class-based bins. "
```

Comment:
"Define which method BiG-SCAPE should use to separate BGC records into analysis bins. '--classify class' and '--classify category' use antiSMASH/BGC classes (e.g. T2PKS) or categories (e.g. PKS) to run analyses on class/category-based bins, respectively.

'--classify legacy' which is based on BiG-SCAPE v1 predefined groups: PKS1, PKSOther, NRPS, NRPS-PKS-hybrid, RiPP, Saccharide, Terpene, Others, and will automatically use complementary '--legacy_weights'. This feature is available for backwards compatibility with input .gbks generated with antiSMASH versions up to version 7. For higher antiSMASH versions, use at your own risk, as BGC classes may have changed. All antiSMASH classes that this legacy mode does not recognise will be grouped in 'others'.

'--classify class' and '--classify category' can be used in combination with --legacy_weights if input .gbks have been generated by antiSMASH version 6 or higher. For older antiSMASH versions, either use --classify "legacy" or do not select --legacy_weights, which will perform the weighted distance calculations based on the generic 'mix' weights. For more detail, see wiki. (default: class)